File: weeks/02/7_mars_exploration/mars_exploration.js
function marsExploration(s) {
  const CORRECT_MESSAGE = "SOS";
  let changedLetters = 0;

  for (let i = 0; i < s.length; i++) {
    if (s[i] !== CORRECT_MESSAGE[i % CORRECT_MESSAGE.length]) {
      changedLetters++;
    }
  }

  return changedLetters;
}

File: weeks/02/7_mars_exploration/mars_exploration.cpp
int marsExploration(string s) {
    const std::string CORRECT_MESSAGE = "SOS";
    int changedLetters = 0;

    for (size_t i = 0; i < s.length(); i++) {
        if (s[i] != CORRECT_MESSAGE[i % CORRECT_MESSAGE.length()]) {
            changedLetters++;
        }
    }

    return changedLetters;

}
